src/config.js

```diff
@@ -2,15 +2,21 @@ const dotenv = require('dotenv');
 
 const port = '8080';
 const host = 'localhost';
-const staticPath = '../resources/html'
+const staticPath = '../resources';
+const accessType = 'Bearer';
+const accessToken = '';
+const apiBaseURL = 'https://graph.microsoft.com/beta';
 
 dotenv.config();
 
 const config = () => {
     return {
         port: process.env.PORT? process.env.PORT : port,
         host: process.env.HOST? process.env.HOST : host,
-        staticPath: process.env.STATIC_PATH? process.env.STATIC_PATH : staticPath
+        staticPath: process.env.STATIC_PATH? process.env.STATIC_PATH : staticPath,
+        accessType: process.env.OAUTH_ACCESS_TYPE? process.env.OAUTH_ACCESS_TYPE : accessType,
+        accessToken: process.env.OAUTH_ACCESS_TOKEN_USER? process.env.OAUTH_ACCESS_TOKEN_USER : accessToken,
```

Comment:
I think the `Bearer` is the only possible value. Do we really need to allow configuring it?